terraform/cloudfoundry/network_acls_cf_in.tf

```diff
@@ -123,7 +123,7 @@ resource "aws_network_acl_rule" "100_ssh" {
     rule_number = 100
     rule_action = "allow"
     from_port = 22
-    to_port = 22
+    to_port = 9300
```

Comment:
Personally I'd prefer the limit being raised, but yes changing the syslog one is a much less terrible solution! Good call.